lib/components/AddDownloadLocationScreen/custom_download_location_form.dart

```diff
@@ -109,14 +107,14 @@ class _CustomDownloadLocationFormState
               }
 
               // This checks if the chosen directory is empty
-              if (selectedDirectory!
+              /*if (selectedDirectory!
                   .listSync()
                   .where((event) => !event.path
                       .replaceFirst(selectedDirectory!.path, "")
                       .contains("."))
                   .isNotEmpty) {
                 return AppLocalizations.of(context)!.directoryMustBeEmpty;
-              }
+              }*/
```

Comment:
I moved to putting everything inside a 'finamp' folder when using custom download locations, so it's probably fine to remove it because we aren't putting a whole bunch of folders in like we did before.  Or maybe I should go back to that?

Reply:
No that should work. If people mess it up, we can add a disclaimer when adding custom locations that warns people about tampering with the downloads directory